UnitTests/BraintreeCoreTests/MockPresentationContext.swift

```diff
@@ -0,0 +1,7 @@
+import AuthenticationServices
+
+class MockPresentationContext: NSObject, ASWebAuthenticationPresentationContextProviding {
+    func presentationAnchor(for session: ASWebAuthenticationSession) -> ASPresentationAnchor {
+        return ASPresentationAnchor()
```

Comment:
I think we can just use the implicit return here:
```suggestion
        ASPresentationAnchor()
```

Reply:
Updated: [dc4e780](https://github.com/braintree/braintree_ios/pull/1598/commits/dc4e780fa5f50b955561bc99a11adfb020fda333)